Ludown/lib/toLU-helpers.js

```diff
@@ -16,20 +16,20 @@ const toLUHelpers = {
         let fileContent = '';
         let luisObj = new helperClasses.rLuisObj();
         let intentInObj;
-        if(LUISJSON.intents.length >= 0) {
+        if(LUISJSON.intents && LUISJSON.intents.length >= 0) {
             LUISJSON.intents.forEach(function(intent) {
                 luisObj.intents.push(new helperClasses.intent(intent, []));
             });
         }
-        if(LUISJSON.utterances.length >= 0) {
+        if(LUISJSON.utterances && LUISJSON.utterances.length >= 0) {
             LUISJSON.utterances.forEach(function(utteranceObj) {
                 intentInObj = luisObj.intents.filter(function(item) {
```

Comment:
This is filtering on each iteration but the results of each filter are not aggregated since the `intentObj` is simply being overwritten. It seems like this either: 1. is erroneous code or 2. loops more than it needs to.